website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -84,6 +84,12 @@ $ export CONSUL_HTTP_TOKEN=$(kubectl get secret consul-bootstrap-acl-token --tem
 
 ## Register external services with Consul
 
+<Note>
+
+Consul on Kubernetes now supports the `Registration` CRD to register services running on external nodes without a terminating gateway. We recommend registering services with this CRD instead of deploying a terminating gateway. For more information, refer to [Register services running on external nodes to Consul on Kubernetes](/consul/docs/k8s/deployment-configurations/external-service).
```

Comment:
should this say `register service running on external nodes with a terminating gateway`? I'm not sure this note makes sense here as we still need a terminating gateway to access those services, the `Registration` CRD just makes registering those services for use with a terminating gateway easier